spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1756,4 +1756,21 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("get_struct_field") {
+    withSQLConf(
+      CometConf.COMET_ROW_TO_COLUMNAR_ENABLED.key -> "true",
+      CometConf.COMET_ROW_TO_COLUMNAR_SUPPORTED_OPERATOR_LIST.key -> "FileSourceScan") {
+      withTempPath { dir =>
+        var df = spark
+          .range(5)
+          .select(struct(col("id")).alias("nested"))
```

Comment:
Do you mind adding a null row (the struct is null) to this test case?

Reply:
Hah I did have that originally but it looked like most things just relied on the Spark tests for extensive testing, I can add back in some nulls

Reply:
Added